requirements/edx/github.in

```diff
@@ -89,7 +89,6 @@
 # Our libraries:
 -e git+https://github.com/edx/codejail.git@a320d43ce6b9c93b17636b2491f724d9e433be47#egg=codejail
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@2.1.17#egg=ora2==2.1.17
 -e git+https://github.com/edx/RecommenderXBlock.git@1.3.3#egg=recommender-xblock==1.3.3
```

Comment:
```suggestion
# Edraak override
-e git+https://github.com/Edraak/edx-ora2.git@v2.1.18#egg=ora2==2.1.18

-e git+https://github.com/edx/RecommenderXBlock.git@1.3.3#egg=recommender-xblock==1.3.3
```

Reply:
Please override on the same line. It makes git conflict resolution easier to reason about.

Reply:
Done

Reply:
Done